daisy_workflows/build-publish/rhui/install_rhua.sh

```diff
@@ -105,6 +105,8 @@ for unit in rhui-health-check.{service,timer}; do
   install -m 664 -t /etc/systemd/system $tempdir/$unit
   systemctl enable $unit
 done
+( cd /etc/nginx; patch -b < $tempdir/nginx.conf.patch; )
```

Comment:
do you think we should follow our existing paradigm and patch the ansible template, instead?

Reply:
I'm not sure of the benefit in this case. Were you thinking of inserting a step for [ansible.posix.patch](https://docs.ansible.com/ansible/latest/collections/ansible/posix/patch_module.html) after this step? 

```
    - name: Provide NginX Pulp specific configuration
      template:
        src: "{{ template_prefix }}/nginx-pulp.conf"
        dest: /etc/nginx/nginx.conf
        owner: root
        group: root
        mode: 0644
      notify: Restart NginX
```

What benefits were you thinking?

Reply:
mainly consistency, so we can see all the changes we're making in one spot. i think we will want to ask for this particular change to be included in the upstream ansible template, too

Reply:
OK, sgtm. Updated to use ansible for patching `nginx-pulp.conf`